src/ui/server/controllers/insurance/eligibility/buyer-country/index.ts

```diff
@@ -90,13 +90,9 @@ export const post = async (req: Request, res: Response) => {
       return res.redirect(CANNOT_APPLY_ROUTE);
     }
 
-    /**
-     * If a country has no insurance support and no short term cover,
-     * redirect to CONTRACT_TOO_SHORT_EXIT.
-     */
-    const noShortTermCover = !country.noInsuranceSupport && !country.shortTermCover;
+    if (country.noOnlineInsuranceSupport) {
```

Comment:
do we need to add any documentation? it is pretty self explanatory tbh but would be nice to have

Reply:
Have added to all country risk redirection conditions